site/en/docs/webstore/google-analytics/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Using your Google Analytics account with the Chrome Web Store"
+seoTitle: "Using your Google Analytics account with the Chrome Web Store"
+date: 2023-07-11
+description: The Chrome Web Store offers integration with Google Analytics, which allows you to see analytics for your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.
+---
+
+The Chrome Web Store offers integration with Google Analytics. This allows you to see analytics for
+your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.
+
+If you’re instead looking to track usage of your extension, see
+[Using Google Analytics 4][extensions-ga4].
+
```

Comment:
Should this have some background on why developers need to do this?

You can get some information from the [MSA we sent out](https://docs.google.com/document/d/1RHxo1dMGGvFD7x-Ml77cu3CB-LE03EIBCK_SwCA8rU4/edit). Please don't write it like an advertisement for the new shiny (as we did in the MSA). Language like that will just sound weird a year from now.

Reply:
I don't think so (going to resolve for now), but happy to chat more and update if needed. I specifically didn't include anything like that to avoid making this specific to the current transition. This page hopefully just answers the question "How do I see who's finding my extension" without making any reference to the history.